scripts/install.sh

```diff
@@ -208,6 +212,15 @@ function init() {
     exportKubeconfig
     KUBEADM_TOKEN_CA_HASH=$(cat /tmp/kubeadm-init | grep 'discovery-token-ca-cert-hash' | awk '{ print $2 }' | head -1)
 
+    if [ "$CIS_COMPLIANCE" == "1" ]; then
+        kubectl apply -f $kustomize_kubeadm_init/pod-security-policy-privileged.yaml
+        # patch 'PodSecurityPolicy' to kube-apiserver and wait for kube-apiserver to reconcile
+        old_admission_plugins='--enable-admission-plugins=NodeRestriction'
+        new_admission_plugins='--enable-admission-plugins=NodeRestriction,PodSecurityPolicy'
+        sed -i "s%$old_admission_plugins%$new_admission_plugins%g"  /etc/kubernetes/manifests/kube-apiserver.yaml
+        wait_for_kube_apiserver
```

Comment:
is there a race condition here?
What happens if the pod hasn't gone down yet before the test starts?

Reply:
Once we patch the kube-apiserver, the reconciler will instantly bring the pod down as it tries to match the new kube-apiserver spec and create a new pod/container.
In, wait_for_kube_apiserver there is kube-apiserver to send requests to, it will timeout and keep on retrying till the kube-apiserver is up and running.

I didn't see a latency issue when testing, but if the kube-apiserver isn't reconciled quickly, the wait_for_kube_apiserver may return old pod status.
Should we add a sleep 1 or sleep 2 to cover the latency???